sap/xssec/security_context_xsuaa.py

```diff
@@ -73,7 +73,7 @@ def _delete_cert_and_key_files(cert_file_name, key_file_name):
         unlink(key_file_name)
 
 
-class SecurityContext(object):
+class SecurityContextXSUAA(object):
```

Comment:
Hi @allenlkq,
Just and idea, I know this was based on the nodejs version but it wouldn't be better to keep `SecurityContext` here and just add `SecurityContextIAS` as a new security context? My idea is to stay backwards compatible, because otherwise it's a breaking change.

Reply:
I have added `security_context.SecurityContext` back as an alias to `security_context_xsuaa.SecurityContextXSUAA`

Reply:
super! thanks!